lib/toaster/bldcollector/views.py

```diff
@@ -0,0 +1,44 @@
+#
+# BitBake Toaster Implementation
+#
+# Copyright (C) 2014        Intel Corporation
+#
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+from django.urls import reverse
+from django.http import HttpResponseBadRequest, HttpResponse
+import os
+import tempfile
+import subprocess
+import toastermain
+from django.views.decorators.csrf import csrf_exempt
+
+
+@csrf_exempt
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Disabling Cross-Site Request Forgery (CSRF) protection can make your users vulnerable to attacks. Enable CSRF if your application can be accessed via a browser and another token mechanism isn&#39;t in place.

[Learn more](https://cwe.mitre.org/data/definitions/352.html)

Similar issue at line numbers 19, 21, 22, 25, 26, 27, 28, 29, 33, and 44.